pages/developer/other_commands.md

```diff
@@ -6,7 +6,7 @@ navigation_source: docs_nav
 
 ## Installing the latest SemVer-compatible version of everything
 
-Normally `rush update` only makes the minimal incremental changes necessary to satisfy the the project **package.json** files.  If you want to update everything to the latest version, you would do this:
+Normally `rush update` only makes the minimal incremental changes necessary to satisfy the project **package.json** files.  If you want to update everything to the latest version, you would do this:
```

Comment:
@Artak-Mkrtchyan Thanks for making this PR. We overlooked it because the Rush website has been moved to a new monorepo.  I have ported your fix to this PR in the new repo: https://github.com/microsoft/rushstack-websites/pull/82